funding/manager.go

```diff
@@ -806,19 +808,20 @@ type PendingChanID = [32]byte
 func (f *Manager) nextPendingChanID() PendingChanID {
 	// Obtain a fresh nonce. We do this by encoding the current nonce
 	// counter, then incrementing it by one.
-	f.nonceMtx.Lock()
-	var nonce [8]byte
-	binary.LittleEndian.PutUint64(nonce[:], f.chanIDNonce)
-	f.chanIDNonce++
-	f.nonceMtx.Unlock()
+	nextNonce := f.chanIDNonce.Add(1)
```

Comment:
It looks like we're changing the behavior here. Before, we encoded the _current_ value, now we're encoding the _next_ value. Do these values need to be in sync between node implementations? Or is this value only used locally?

Reply:
Discussed offline, this is fine.